Back/src/app.service.ts

```diff
@@ -0,0 +1,8 @@
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class AppService {
+  getHello(): string {
+    return 'Hello World!';
+  }
+}
```

Comment:
Service should containe the business logic regarding the entity they represent. One of the advantages of prisma is taht this is done automatically for you so we can save time just putting the prisma service directly at the service layer.